benchmark/child_process/child-process-read.js

```diff
@@ -19,7 +19,7 @@ function main(conf) {
   const dur = +conf.dur;
   const len = +conf.len;
 
-  const msg = '"' + Array(len).join('.') + '"';
+  const msg = `"${'.'.repeat(len)}"`;
```

Comment:
Yes, in the strict sense. But it is not a test, so there are no assertions or checks. It seems, the intention was the length should be `[64, 256, 1024, 4096]`, just `+ 1` was not bothered to be added. Or do I miss something?